src/redisWrapper.js

```diff
@@ -14,19 +14,22 @@ const COMPONENT_NAME = "/RedisWrapper";
 const VERROR_CLUSTER_NAME = "RedisWrapperError";
 
 const INTEGRATION_MODE = Object.freeze({
+  CF_REDIS_CLUSTER: "CF_REDIS_CLUSTER",
   CF_REDIS: "CF_REDIS",
   LOCAL_REDIS: "LOCAL_REDIS",
   NO_REDIS: "NO_REDIS",
 });
 
 const logger = new Logger(COMPONENT_NAME);
+const redisCredentials = cfEnv.cfServiceCredentialsForLabel("redis-cache");
 
 const MODE = Object.freeze({
   RAW: "raw",
   OBJECT: "object",
 });
 
 let redisIsOnCF = isOnCF;
+let redisIsCluster = redisCredentials.cluster_mode;
```

Comment:
same here

Reply:
This is why the other part is top level. I don't want to move this in the cf env class because it only exists in the redis credentials and is not cf specific.